### PR TITLE
Fix all flake8 errors except E722 (bare excepts)

### LINKED ISSUE #16: Fix all flake8 issues

### PR DESCRIPTION
**Pull request by SkullTech**
Fixes all linting errors (flake8) except E722, which prevents bare exceptions. Fixes #16 

I think we should ignore E722 in `setup.cfg`, because as of now, they are unavoidable in some parts of the code.

### CODE REVIEW

**Comment by marco-c** [comment]
Can you also rebase on top of master, so we see the Travis build becoming green? :)

File: utils.py [hunk 1]
```diff
@@ -4,7 +4,7 @@
 import numpy as np
 from PIL import Image
 import keras
-from keras.preprocessing.image import ImageDataGenerator, Iterator, load_img, img_to_array
+from keras.preprocessing.image import ImageDataGenerator, load_img, img_to_array  # , Iterator
```
Comment (marco-c) [inline utils.py]: You can remove `Iterator` altogether here.

**Comment by SkullTech** [comment]
Did as you suggested. Didn't get what you meant about rebasing on top of mastering though, is that necessary? My branch is going to merge with the `main` branch of this repo anyway.

**Comment by marco-c** [comment]
> Did as you suggested. Didn't get what you meant about rebasing on top of mastering though, is that necessary? My branch is going to merge with the main branch of this repo anyway.

I meant `git pull upstream master --rebase`, but it's fine, don't worry ;)

**Pull request merged**